spec.html

```diff
@@ -47259,7 +47259,7 @@ <h1>Properties of Promise Instances</h1>
     <h1>GeneratorFunction Objects</h1>
     <p>GeneratorFunctions are functions that are usually created by evaluating |GeneratorDeclaration|s, |GeneratorExpression|s, and |GeneratorMethod|s. They may also be created by calling the %GeneratorFunction% intrinsic.</p>
     <emu-figure id="figure-2" caption="Generator Objects Relationships" informative>
-      <img alt="A staggering variety of boxes and arrows." src="img/figure-2.png">
+      <img alt="A staggering variety of boxes and arrows." height="660" src="img/figure-2.svg" width="900">
```

Comment:
i think the width/height info can be inside the svg data, no?

Reply:
The SVG file also contains those info, but having the `width`/`height` attributes helps the layout until the SVG gets loaded.
Other `<img>` elements also specify those attributes.